install.sh

```diff
@@ -13,7 +13,7 @@ function display_help()
     echo "    [-p|--package_build] Build RCCL package."
     echo "    [-t|--tests_build] Build unit tests, but do not run."
     echo "    [-r|--run_tests] Run unit tests (must be built already.)"
-    echo "    [--hip-clang] Build library using hip-clang compiler."
```

Comment:
Can we keep a silent --hip-clang option? Devops passes this to the CI and we don't want to break builds. Once all libraries update and devops fixes their CI, we can remove this.

Reply:
Will do.